asv/util.py

```diff
@@ -531,7 +543,7 @@ def get_content(header=None):
     if env and WIN and sys.version_info < (3,):
         # Environment keys and values cannot be unicode
         def _fix_env(s):
-            return s.encode('mbcs') if isinstance(s, unicode) else s
+            return s.encode('mbcs') if isinstance(s, unicode) else s # noqa
```

Comment:
Another space is missing before the hash. Also, we need the error we are ignoring after noqa.

Reply:
Done!